### PR TITLE
feat: Add prompthub models migration

### PR DESCRIPTION
**Pull request by anticorrelator**

### CODE REVIEW

**Review by RogerHYang — APPROVED** [review]

File: src/phoenix/db/migrations/versions/bc8fea3c2bc8_add_prompt_tables.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        sa.Column("name", sa.String, nullable=False, unique=True),
+        sa.Column("description", sa.String),
+        sa.UniqueConstraint("name", name="uq_prompt_tag_configs_name"),
+    )
+
+    op.create_table(
+        "prompts",
+        sa.Column("id", sa.Integer, primary_key=True),
+        sa.Column("name", sa.String, unique=True, nullable=False),
+        sa.Column("description", sa.String),
```
Comment (mikeldking) [inline src/phoenix/db/migrations/versions/bc8fea3c2bc8_add_prompt_tables.py:60]: Just a note to future us - we probably need to account for if the prompt is "cloned"

File: src/phoenix/db/migrations/versions/bc8fea3c2bc8_add_prompt_tables.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+            nullable=False,
+            server_default=sa.func.now(),
+        ),
+        sa.Column(
+            "updated_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.func.now(),
+            onupdate=sa.func.now(),
+        ),
+        sa.UniqueConstraint("name", name="uq_prompts_name"),
```
[the remark this answers is not in the document]
Comment (anticorrelator) [reply src/phoenix/db/migrations/versions/bc8fea3c2bc8_add_prompt_tables.py:73]: ah right yeah I think they are

File: src/phoenix/db/migrations/versions/bc8fea3c2bc8_add_prompt_tables.py [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+            "user_id",
+            sa.Integer,
+            sa.ForeignKey("users.id", ondelete="SET NULL"),
+            nullable=True,
+            index=True,
+        ),
+        sa.UniqueConstraint(
+            "name",
+            "prompt_id",
+            name="uq_prompt_template_version_tags_name_prompt_id",
```
[the remark this answers is not in the document]
Comment (anticorrelator) [reply src/phoenix/db/migrations/versions/bc8fea3c2bc8_add_prompt_tables.py:174]: I matched the constraint name in the models file for this

[the remark this answers is not in the document]
Comment (anticorrelator) [reply src/phoenix/db/migrations/versions/bc8fea3c2bc8_add_prompt_tables.py:173]: ah I see, I can remove them both

**Pull request merged**